proto/xudrpc.proto

```diff
@@ -241,6 +249,11 @@ message BanRequest {
 }
 message BanResponse {}
 
+message UnBanRequest {
```

Comment:
Oh and one more thing, I've been trying to keep the messages and calls in alphabetic order respectively. Could you move these messages down to the bottom of the file, and also move the `rpc Unban` to the bottom of the call definitions?